draft-ietf-rtcweb-jsep.xml

```diff
@@ -827,8 +827,7 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
         remote endpoint will only send a single encoding per m=
         section. In addition, when the JSEP endpoint is the answerer,
         the permitted encodings for the RTPSender must be consistent
-        with the offer, but this information is currently not surfaced
-        through any API. This means that established simulcast streams
+        with the offer. This means that established simulcast streams
```

Comment:
I don't follow this reasoning, unless you interpret "does not provide an API to configure receipt of simulcast" to mean "is unable to receive an offer to send simulcast", which I have been treating as two separate things.

Regarding receipt of a simulcast offer, the difference between an outgoing simulcast offer and an incoming one is that the application knows what it's trying to do in the first case, and can configure the session appropriately. In the incoming case, the application doesn't know that it's received a simulcast offer. IOW, the lack of inspection capability is material, and the explanation of why this can't work doesn't make much sense without it.